lib/material/internal/material_player.flow

```diff
@@ -1184,16 +1185,18 @@ TAnimatedSubtitles(manager : MaterialManager, parent : MFocusGroup, subtitles :
 		MDynamicParagraph2T(parent, const(sbtl.text), concat(
 			replaceStruct(MCharacterStyle2MTextStyle(sbtl.style), BackgroundFillOpacity(0.0)),
 			[
-				MSetRTL(false),
+				MSetRTL(isRTLLang(getLang())),
 				extractStruct(sbtl.style, EscapeHTML(true)),
-				CenterAlign(),
+				fgetValue(paragraphAlignment),
 				ParagraphMetrics(\met -> nextDistinct(size, WidthHeight(met.width, met.height + dSize)))
 			])
 		)
 		|> TBorderBottom(dSize)
 		|> TBorderTop(frame.border)
 		|> (\t -> TTranslate(translatePoint, t))
-		|> (\t -> TConstruct([\ -> \ -> nextDistinct(size, zeroWH)], t));
+		|> (\t -> TConstruct([\ -> \ -> nextDistinct(size, zeroWH)], t))
+		|> (\t -> TAvailableWidth(t, fwidth(boxSize)))
```

Comment:
It introduces cyclic dependency: boxSize is calculated from line metrics and line metrics are dependent on available width